### PR TITLE
`time-release` mock and unit tests should match runtime configuration

### LINKED ISSUE #1836: `time-release` pallet mock and unit test need updates
In testing it was discovered that the mock runtime value for `MinReleaseTransfers` does not match the value for the runtime, it should be corrected to match.
Tests should be added to make sure that release schedules cannot be added to accounts without an Existential Deposit.

### PR DESCRIPTION
**Pull request by mattheworris**
# Goal
The goal of this PR is to update the `time-release` mock so the the `MinReleaseTransfer` matches the value in the runtime configuration.

This may cause follow on failures in some of the unit tests.

Closes #1836

# Changes
- [x] `MinReleaseTransfer` updated to 0
- [x] Unit tests updated
- [x] `.vscode/settings.json` updated to include more targets for better feature coverage

# How to Test
- Confirm `time-release` unit tests pass

# Checklist
- [ ] Chain spec updated [not needed: no runtime changes]
- [ ] Custom RPC OR Runtime API added/changed? Updated js/api-augment.
- [ ] Design doc(s) updated
- [ ] Tests added
- [ ] Benchmarks added
- [ ] Weights updated

### CODE REVIEW

File: pallets/time-release/src/benchmarking.rs [hunk 1]
```diff
@@ -16,8 +16,8 @@ pub type Schedule<T> = ReleaseSchedule<BlockNumberFor<T>, BalanceOf<T>>;
 const SEED: u32 = 0;
 
 fn set_balance<T: Config>(who: &T::AccountId, balance: BalanceOf<T>) {
-	let actual_deposit = T::Currency::mint_into(&who, balance.saturated_into());
-	assert_eq!(balance, actual_deposit.unwrap());
+	let actual_deposit = T::Currency::set_balance(&who, balance.saturated_into());
```
Comment (mattheworris) [inline pallets/time-release/src/benchmarking.rs:19]: `mint_into` fails when the account has a `0` balance. 
`set_balance` is infallible and provided by polkadot-sdk specifically for benchmarking and testing.

File: .vscode/settings.json [hunk 2]
```diff
@@ -15,9 +15,12 @@
     ],
     // Set the features to use for cargo commands
     "rust-analyzer.cargo.features": [
-        "frequency-no-relay"
+        "frequency-no-relay",
+        "runtime-benchmarks",
```
Comment (mattheworris) [inline .vscode/settings.json:19]: Adding these targets to cover more code, e.g. benchmarking.rs files do not know about any definitions if `runtime-benchmarks` is not defined.

**Review by JoeCap08055 — APPROVED** [review]
Ran unit tests locally; all passed

**Review by enddynayn — APPROVED** [review]
Looks great! 👍

**Review by saraswatpuneet — APPROVED** [review]

**Pull request merged**